.github/workflows/publish.yml

```
@@ -7,37 +7,115 @@ on:
+      - name: Action | Semantic Version Release
+        id: release
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          rm -rf dist/ build/ *.egg-info
+          chmod +x scripts/semantic-version-release.sh
+          ./scripts/semantic-version-release.sh releaserc.toml
+
+      - name: Get | Current Commit SHA
+        id: get-commit
+        run: echo "commit_sha=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:44: Double quote to prevent globbing and word splitting

(shellcheck)